woocommerce/woocommerce
# Move the DatabaseUtil class to the proper directory

### All Submissions:


### Changes proposed in this Pull Request:

The `DatabaseUtil` class is in the `Automattic\WooCommerce\Internal\Utilities` namespace, but it was wrongly placed in the `src/Utilities` directory. It has been moved to `src/Internal/Utilities` without changing the namespace, thus no code changes are required.

Closes https://github.com/woocommerce/woocommerce/pull/32836 (note that it's a pull request, but it just notifies the issue, it doesn't fix it)

### How to test the changes in this Pull Request:

1. Run `composer dump-autoload`
2. Run the following from cli and verify that it works:

```
 wp eval 'var_dump(wc_get_container()->get(Automattic\WooCommerce\Internal\Utilities\DatabaseUtil::class)
->get_index_columns("wp_wc_product_attributes_lookup"));'
```

### Other information:




### FOR PR REVIEWER ONLY:
